npc/events/christmas_2006.txt

```diff
@@ -0,0 +1,4165 @@
+		mes "[Christ]";
+		mes "My Peco is not a male!?Even though she can't lay eggs,it is still a female. Apologize to her!";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "...I'm sorry.";
+		mes "By the way, did Enjoy give you any clue...";
+		next;
+		mes "[Christ]";
+		mes "!!";
+		mes "Enjoy? Then, it must be the time......";
+		next;
+		mes "[Christ]";
+		mes "^0000ffUnder the bench^000000!";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "What do you mean?";
+		next;
+		mes "[Christ]";
+		mes "^0000ffUnder the bench^000000!";
+		next;
+		mes "[Christ]";
+		mes "I'm sorry, that's all I can say.Go ask other members.";
+		next;
+		mes "[Christ]";
+		mes "Well then, I'll meet you 'there'.";
+		05christ_solo = 11;
+		close;
+	} else if (05christ_solo > 10) {
+		mes "[Christ]";
+		mes "^0000ffUnder the bench^000000!";
+		next;
+		mes "[Christ]";
+		mes "I'm sorry, that's all I can say.Go ask other members.";
+		next;
+		mes "[Christ]";
+		mes "Well then, I'll meet you 'there'.";
+		close;
+	} else {
+		mes "[Christ]";
+		mes "Pecopeco~! Where should we head to??We've always been together and had overcome all difficulties and always will. Aren't we?";
+		close;
+	}
+}
+
+prontera,163,66,7	script	Mas#06xmas_01	4_M_ALCHE_A,{
+	if (05christ_solo == 11) {
+		mes "[Mas]";
+		mes "Herbs,potions.I can make it all by myself!!All I need is an honour.";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "Are you Mas?";
+		next;
+		mes "[Mas]";
+		mes "Who....Who.. are...you?!";
+		mes "I...I...I don't have time...Do not disturb me!!";
+		next;
+		mes "[Mas]";
+		mes "... hum..it feels good talking to someone..But no no no no!! I'm running out of time!Go away!!";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "Hey!Wake up!!";
+		mes "I was just wondering whether Enjoy have left a...";
+		next;
+		mes "[Mas]";
+		mes "Enjoy? Did you just say Enjoy?";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "Yep! didn't Enjoy say anything to you?...";
+		next;
+		mes "[Mas]";
+		mes "!!";
+		mes "Oh, I didn't realize how fast has the time flown...I was digging herds and making a potions everyday.....";
+		next;
+		mes "[Mas]";
+		mes "^0000ffWhere the sun goes down^000000!";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "What did you say?";
+		next;
+		mes "[Mas]";
+		mes "^0000ffWhere the sun goes down^000000!";
+		next;
+		mes "[Mas]";
+		mes "I'm sorry, that's all I know.Go ask other members.";
+		next;
+		mes "[Mas]";
+		mes "Well then, I'll meet you 'there'.";
+		05christ_solo = 12;
+		close;
+	} else if (05christ_solo > 11) {
+		mes "[Mas]";
+		mes "^0000ffWhere the sun goes down^000000!";
+		next;
+		mes "[Mas]";
+		mes "I'm sorry, that's all I know.Go ask other members.";
+		next;
+		mes "[Mas]";
+		mes "Well then, I'll meet you 'there'.";
+		close;
+	} else {
+		mes "[Mas]";
+		mes "Herbs,potions.I can make it all by myself!! All I need is an honour.";
+		close;
+	}
+}
+
+prontera,35,209,5	script	Event#06xmas_01	4_M_SITDOWN,{
+	if (05christ_solo == 12) {
```

Comment:
`.@name$ = strcharinfo(0);` to avoid the redundant command call

Reply:
Applied suggested fix.